### ISSUE TITLE
website: redirect cuelang.org/cl like we do for cuelang.org/issue

### ISSUE THREAD

**Issue by mvdan** [cue-lang/cue#1721]
URLs like https://cuelang.org/issue/123 redirect to our issue tracker, currently https://github.com/cue-lang/cue/issues. It helps to use URLs like that because, if we ever need to move the issue tracker, the URLs can be redirected accordingly.

We should do the same for CLs; for example, https://cuelang.org/cl/538063 is just a 404, when it could redirect to https://review.gerrithub.io/c/cue-lang/cue/+/538063. As an extra bonus, note how much longer the real Gerrit URL is, so that will also help for commit messages.

**Comment by cedricgc** [comment]
@mvdan I see the GH issue redirect is configured in the netlify config here:
https://github.com/cue-lang/cuelang.org/blob/de33ccc1e61eb85265682179c9af8bd710b482c5/netlify.toml#L15

Does this issue make more sense in the cue-lang/cuelang.org repo? I can take a crack at the implementation, though I think the redirect can only be tested post-merge and deploy. I would need to dig into the netlify docs.

**Comment by mvdan** [comment]
@cedricgc you're absolutely right about this being the wrong repository :) I think this is just my habit since Go uses a single issue tracker for their multiple repos, but we don't here.
